cmake/gulrak.cmake

```diff
@@ -26,23 +24,18 @@ if(DEFINED GULRAK_SOURCE_DIR)
     file(TO_CMAKE_PATH "${GULRAK_SOURCE_DIR}" GULRAK_SOURCE_DIR)
     message(STATUS "**** Building Gulrak From " ${GULRAK_SOURCE_DIR})
     FetchContent_Declare(
-        ${CONTENT_NAME}
+        gulrak
         URL ${GULRAK_SOURCE_DIR}
     )
 
-    string(TOUPPER ${CONTENT_NAME} UPPERGULARK)
-    mark_as_advanced(FETCHCONTENT_SOURCE_DIR_${UPPERGULARK})
+    mark_as_advanced(FETCHCONTENT_SOURCE_DIR_GULRAK)
     mark_as_advanced(FETCHCONTENT_UPDATES_DISCONNECTED_${UPPERGULARK})
-
 else()
     message(STATUS "**** Building Gulrak From Github Repository.")
     FetchContent_Declare(
-        ${CONTENT_NAME}
-        GIT_REPOSITORY https://github.com/gulrak/filesystem.git
-        GIT_TAG        4e21ab305794f5309a1454b4ae82ab9a0f5e0d25
-        USES_TERMINAL_DOWNLOAD TRUE
-        GIT_CONFIG     advice.detachedHead=false
+        gulrak
+        URL https://github.com/gulrak/filesystem/archive/refs/tags/v1.5.14.zip
```

Comment:
The previous version was 1.5.6 (which is what that git commit pointed to) which seems like a big jump to 1.5.14, but its not that really. There weren't all the dot releases in between. https://github.com/gulrak/filesystem/releases

I also changed it so that it just downloads the release zip file instead of cloning the github repo.